utils/test/clearing_operations_tests/ClearingOperationsCreateGRGFromAreaTestCase.py

```diff
@@ -88,14 +87,14 @@ def testClearingOperationsAreaGRG(self):
         arcpy.AddMessage(runToolMsg)
         Configuration.Logger.info(runToolMsg)
 
-        try:
-			# Calling the CanvasAreaGRG_ClearingOperations Script Tool
-            arcpy.CanvasAreaGRG_ClearingOperations(self.inputArea, cellWidth, cellHeight, cellunits, labelStart, labelStyle, output)
-			
-        except arcpy.ExecuteError:
-            UnitTestUtilities.handleArcPyError()
-        except:
-            UnitTestUtilities.handleGeneralError()
+        # try:
+        # Calling the Create GRG From Area script tool
+        arcpy.CreateGRGFromArea_clrops(self.inputArea, cellWidth, cellHeight, cellunits, labelStart, labelStyle, output)
+
+        # except arcpy.ExecuteError:
+        #     UnitTestUtilities.handleArcPyError()
```

Comment:
Note: these methods: handleArcPyError, handleGeneralError were changed to re-throw the exception (so tests will fail as expected on exception) so you probably don't need to comment them out any more

Reply:
Done.